src/Rules.Framework/Extensions/GenericRuleExtensions.cs

```diff
@@ -18,8 +18,9 @@ public static GenericConditionNode ToGenericConditionNode<TConditionType>(this I
                 {
                     ConditionTypeName = condition.ConditionType.ToString(),
                     DataType = condition.DataType,
+                    LogicalOperator = condition.LogicalOperator,
                     Operand = condition.Operand,
-                    Operator = condition.Operator
+                    Operator = condition.Operator,
```

Comment:
It is because of this static analysis warning.

![image](https://user-images.githubusercontent.com/20577216/229245811-4e24f167-eb56-498d-ae23-bb832560f2fc.png)

We can disable it if you don't agree with this code-style guide, it fairly simple to do so 👌 and then I remove the commas

Reply:
I see no advantage in following that code-style guideline (maybe only when doing git diff if a new assignment is added), and it seems that there is something missing after the comma.

Not really sure why that is a guideline 🤷🏼‍♂️ , but I'm ok with leaving it like that.

Reply:
We did not set a guideline among us actually, I did add this package `Meziantou.Analyzer` which enforces some styling and (more importantly) detects some code smells.

I am totally OK with changing this, I am neither the guy to include the comma, nor the guy to not include it, I am the guy to follow a pattern, that's what's more important to me above all.

Reply:
I will change this, and probably make this a theme of one of our meetings. Can we leave this be for this pull request and resolve on a separate pull request (that way we reflect the change on every place)?

Reply:
Sure, let's do it that way 👌🏼 .